### DISCUSSION TITLE
想通过https://forestry.io/写文章

### DISCUSSION THREAD

**Discussion by waimao365** [CaiJimmy/hugo-theme-stack#327] [Q&A]
想通过https://forestry.io/写文章，但https://forestry.io/的格式是TOML 格式的，而hugo是YAML格式的，有什么办法把huog默认的yaml改成toml格式吗？

**Comment by zhixuan2333** [comment]
https://forestry.io/docs/faqs/how-to-use-yaml-json-front-matter-w-hugo/
随便看了一下文档，这个可能会解决你的问题。
